### PR TITLE
WT-6723 Remove lookaside comments

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: src/txn/txn.c [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ __txn_commit_timestamps_assert(WT_SESSION_IMPL *session)
             continue;
         }
 
-        /* Search for prepared updates, so that they will be restored, if moved to lookaside. */
+        /* Search for prepared updates. */
```
Comment (hbokhari) [inline src/txn/txn.c:1158]: I think it is better to keep the complete sentence and just replace `lookaside` by `history store`.

File: test/suite/test_gc01.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def check_gc_stats(self):
         self.assertGreaterEqual(c[stat.conn.cc_pages_removed][2], 0)
         c.close()
 
-# Test that checkpoint cleans the obsolete lookaside pages.
+# Test that checkpoint cleans the obsolete the history store pages.
```
Comment (hbokhari) [inline test/suite/test_gc01.py:82]: Remove the extra word "the" before history store.

**Review by hbokhari — CHANGES_REQUESTED** [review]

File: src/txn/txn.c (hunk 1 above)
Comment (quchenhao) [reply src/txn/txn.c:1158]: No. then that is wrong. We are not searching the history store here. Before with lookaside, we need to reconstruct the update chain from data in the lookaside but now we only search data store.

**Review by hbokhari — APPROVED** [review]
LGTM

**Pull request merged**